.vib/suitecrm/vib-verify.json

```diff
@@ -17,13 +17,58 @@
       ]
     },
     "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/suitecrm"
+        },
+        "runtime_parameters": "c2VydmljZToKICB0eXBlOiBMb2FkQmFsYW5jZXIKICBwb3J0czoKICAgIGh0dHA6IDgwCiAgICBodHRwczogNDQzCnN1aXRlY3JtSG9zdDogYml0bmFtaS1zdWl0ZWNybS5teQpzdWl0ZWNybVVzZXJuYW1lOiB1c2VyCnN1aXRlY3JtUGFzc3dvcmQ6IENvbXBsaWNhdGVkUGFzc3dvcmQxMjMhNApzdWl0ZWNybVNtdHBIb3N0OiB0ZXN0aG9zdApzdWl0ZWNybVNtdHBQb3J0OiA5MTI1CnN1aXRlY3JtU210cFVzZXI6IHRlc3RIb3N0VXNlcgpzdWl0ZWNybVNtdHBQYXNzd29yZDogdGVzdEhvc3RQYXNzd29yZA==",
```

Comment:
I checked it and this asset indeed requires setting the host; it is one normally installed with a two-step process. You are not encountering any issues because it is accessible in localhost, though it is strange VIB is not throwing a fit. Nevertheless, we should configure Cypress to test it using the host as we do for others like ghost or discourse. For that, you will need to modify the `cypress.json` and `support/commands.js` files to add the host and the `overwrite` command.

Reply:
@FraPazGal If it's accessible in localhost then why do we need to overwrite it with the host?

Reply:
As discussed in the related Slack thread, I've applied this change and it works properly. Thanks for the suggestion!

Reply:
Nice! It is indeed strange that it works using localhost. It could be caused by some chart logic not present in other charts, though we would need to investigate further to know for certain. As using the host in Cypress conf is an approach that works for every instance, I think investigating this is not a priority.